pyfar/dsp/dsp.py

```diff
@@ -187,3 +187,112 @@ def spectrogram(signal, dB=True, log_prefix=20, log_reference=1,
     times -= times[0]
 
     return frequencies, times, spectrogram
+
+
+def normalize(signal, norm_type='time', operation='max', channelwise='max',
+              value=None, freq_range=(20, 22e3)):
```

Comment:
A few more thoughts on more verbose and intuitive parameter names and defaults (sorry I did not think of this before):
- `norm_type` could be `normalize` with the options `time`, `magnitude`, `log_magnitude`
- `operation` could be `normalize_to`
- `channelwise` could be `channel_handling`
- a better default for `freq_range` might be `None` with this later in the code
```
if freq_range is None:
    freq_range = (0, signal.frequencies[-1])
```